src/components/ContextMenu/index.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { StoryObj } from '@storybook/react';
+import ContextMenu, { ContextMenuProps } from '.';
+import { ContextMenuItemProps } from './ContextMenuItem';
+import { ContextMenuContentProps } from './ContextMenuContent';
+
+type AllEvents = Pick<ContextMenuProps, 'onOpenChange'> &
+  Pick<ContextMenuItemProps, 'onSelect'> &
+  Pick<
+    ContextMenuContentProps,
+    | 'onCloseAutoFocus'
+    | 'onEscapeKeyDown'
+    | 'onPointerDownOutside'
+    | 'onFocusOutside'
+    | 'onInteractOutside'
+  >;
+
+const hidden = {
+  table: {
+    disable: true,
+  },
+};
+
+const events = {
+  content: {
+    onCloseAutoFocus: {
+      ...hidden,
+      action: 'onCloseAutoFocus',
+    },
+    onEscapeKeyDown: {
+      ...hidden,
+      action: 'onEscapeKeyDown',
+    },
+    onPointerDownOutside: {
+      ...hidden,
+      action: 'onPointerDownOutside',
+    },
+    onFocusOutside: {
+      ...hidden,
+      action: 'onFocusOutside',
+    },
+    onInteractOutside: {
+      ...hidden,
+      action: 'onInteractOutside',
+    },
+  },
```

Comment:
This all seems very... redundant? I don't necessarily want us to make any function here, but I wonder if you could like show this to some online Storybook people and see if there's any improvements to this sort of boilerplate we could make?

Reply:
i can improve this later, i don't think it should be a blocker for now